src/blocks/social-profiles/test/social-profiles.cypress.js

```diff
@@ -247,8 +247,8 @@ describe( 'Test CoBlocks Social Profiles Block', function() {
 		helpers.openSettingsPanel( 'Icon settings' );
 		helpers.toggleSettingCheckbox( 'Social colors' );
 
-		cy.get( 'button[aria-label="Add Facebook profile"]' )
-			.should( 'have.css', 'background-color', 'rgb(49, 55, 60)' );
+		cy.get( '.block-editor-writing-flow button[aria-label="Add Facebook profile"]' )
+			.should( 'have.css', 'background-color', 'rgb(200, 106, 25)' );
```

Comment:
Here's that variable again, in another file. I don't know this code, but could we consider having a shared file with variables used? We could then only update this file if anything change instead of all the tests.

Reply:
You make a good point with these variables Jason. Something that we need to change regarding E2E testing on CoBlocks is to improve theme agnosticism. The changes in these tests to differing RGB values are a direct result of changing the testing framework from using the TwentyTwenty theme to using the Go theme. Essentially we need to come up with an approach which will work across themes.